### PR TITLE
[ENG-3105] Bulk upload widget, with acceptance tests.

### PR DESCRIPTION
**Pull request by adlius**
-   Ticket: [ENG-3105]
-   Feature flag: n/a

## Purpose

This PR enables admins of registration providers to upload a CSV file and create registrations in bulk.

## Summary of Changes

PLACEHOLDER

## Screenshot(s)

Screenshot from 2021-08-23 12-43-36

## QA Notes

[ENG-3105]: https://openscience.atlassian.net/browse/ENG-3105

### CODE REVIEW

File: lib/registries/addon/branded/moderation/-components/upload-csv/template.hbs [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        @options={{this.dropzoneOptions}}
+        @dropzone={{false}}
+        @addedfile={{action this.addedFile}}
+        @uploadprogress={{action this.uploadProgress}}
+        @success={{action this.success}}
+        @error={{action this.error}}
+        @dragenter={{action (mut this.dropping) true}}
+        @dragover={{action (mut this.dropping) true}}
+        @drop={{action (mut this.dropping) false}}
+        @dragleave={{action (mut this.dropping) false}}
```
Comment (fabmiz) [inline lib/registries/addon/branded/moderation/-components/upload-csv/template.hbs:13]:
```suggestion
        @buildUrl={{this.buildUrl}}
        @options={{this.dropzoneOptions}}
        @dropzone={{false}}
        @addedfile={{this.addedFile}}
        @uploadprogress={{this.uploadProgress}}
        @success={{this.success}}
        @error={{this.error}}
        @dragenter={{fn (mut this.dropping) true}}
        @dragover={{fn (mut this.dropping) true}}
        @drop={{fn (mut this.dropping) false}}
        @dragleave={{fn (mut this.dropping) false}}
```

**Review by fabmiz — COMMENTED** [review]
Looks great 💯 Minor suggestion. I am also not sure why `yarn.lock` was updated, could you not commit the update.

**Comment by adlius** [comment]
> Looks great  Minor suggestion. I am also not sure why `yarn.lock` was updated, could you not commit the update.

yarn.lock is updated to update `ember-template-lint`'s version. I need the latest version (which includes [this commit](https://github.com/ember-template-lint/ember-template-lint/commit/ee1f2b93b79c6abcdfd809c08a2a63addfe7ab8e)) to allow using variables defined in the template as element id. Otherwise template linting will fail.

**Comment by adlius** [comment]
Closed in favor of #1307

**Pull request closed without merging**